src/osgEarthSymbology/TextSymbol

```diff
@@ -204,6 +204,10 @@ public:
     inline optional<bool>& backEarthCull() { return _backEarthCull; }
     inline const optional<bool>& backEarthCull() const { return _backEarthCull; }
 
+    /** The priority defined between symbols  */
+    optional<NumericExpression>& textPriority() { return _textPriority; }
+    const optional<NumericExpression>& textPriority() const { return _textPriority; }
+    
```

Comment:
**Impossible to understand the difference between TextSymbol::priority() and TextSymbol::textPriority**

replace textPriority by priorityFine everywhere